services/gitlab/gitlab-license.service.js

```diff
@@ -0,0 +1,73 @@
+import Joi from 'joi'
+import { renderLicenseBadge } from '../licenses.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.object({
+  license: Joi.object({
+    name: Joi.string().required(),
+  }).allow(null),
+}).required()
+
+const documentation = `
+<p>
+  You may use your GitLab Project Id (e.g. 13083) or your Project Path (e.g. gitlab-org/gitlab-foss )
+</p>
+`
+const commonProps = {
+  namedParams: {
+    project: 'gitlab-org/gitlab-foss',
+  },
+  documentation,
+}
+
+export default class GitlabLicense extends GitLabBase {
+  static category = 'license'
+
+  static route = {
+    base: 'gitlab/v/license',
+    pattern: ':project+',
+  }
```

Comment:
I think we should include the standard query parameter that allows users to specify the GitLab instance to use. (It looks like you structured the `handle` function to accept this parameter already, but we still need to wire up the route properly to ensure the query param is actually available)

You can see an example of this in the GitLab release badge:
https://github.com/badges/shields/blob/fd13db83bab6450e38245cafb9c81cb6c5291225/services/gitlab/gitlab-release.service.js#L14-L15

https://github.com/badges/shields/blob/fd13db83bab6450e38245cafb9c81cb6c5291225/services/gitlab/gitlab-release.service.js#L39-L43

https://github.com/badges/shields/blob/fd13db83bab6450e38245cafb9c81cb6c5291225/services/gitlab/gitlab-release.service.js#L132-L135

Reply:
done